src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/external/OffenderSearchApiService.kt

```diff
@@ -139,33 +125,95 @@ class OffenderSearchApiService(
     return PrisonersList(emptyList(), 0, 0, sort, 0, false)
   }
 
-  private fun objectMapper(searchList: List<PrisonersSearch>, pathwayView: Pathway?, pathwayStatusToFilter: Status?): List<Prisoners> {
+  fun sortPrisoners(sort: String? = null, offenders: MutableList<Prisoners>) {
```

Comment:
```suggestion
  fun sortPrisoners(sort: String?, offenders: MutableList<Prisoners>) {
```
Not sure you are using this default